sormas-ui/src/main/java/de/symeda/sormas/ui/utils/CommitDiscardWrapperComponent.java

```diff
@@ -943,6 +929,10 @@ public CommitRuntimeException(CommitException e) {
 		}
 	}
 
+	public static boolean isEditOrDeleteAllowed(boolean isEditAllowed, boolean isDeleteAllowed) {
```

Comment:
Remove this method, please, and directly use `isEditAllowed || isDeleteAllowed` in all instances where this is called; I don't think it's necessary to move such a trivial logic to a dedicated method, and I don't think that CommitDiscardWrapperComponents hould have a static method like this because it's not intended to be a helper class of any kind (i.e. if we would preserve the method, it would need to be moved somewhere else).